README.md

```diff
@@ -1,21 +1,61 @@
 # scalar-admin
 
-scalar-admin is an admin interface and client tool for Scalar services such as [Scalar DL](https://github.com/scalar-labs/scalardl) and [Scalar DB](https://github.com/scalar-labs/scalardb) applications.
+scalar-admin is an admin interface and client tool for Scalar services such as [Scalar DL](https://github.com/scalar-labs/scalardl) and [Scalar DB](https://github.com/scalar-labs/scalardb) applications. 
+It will help you to `PAUSE` and `UNPAUSE` the ledger to create point-in-time recovery (PITR). 
 
-## Use the client-side tool
+## Client-side tool
+
+The scalar-admin client tool is used to manage the admin interface implemented applications.
+It will help to `PAUSE` and `UNPAUSE` the application without losing the transaction.
+
+### Installation
+
+You can create a build for the scalar-admin client tool using the following methods
+
+#### Jar
```

Comment:
I think we need to show how to install via Docker. 
Like I said, most users just use the container instead of building it.

Reply:
I would like to release a scalar-admin docker image in the scalar docker registry.
So the users can use it like this
```
docker run -it --rm ghcr.io/scalar-labs/scalar-admin:<version> -c <Command> -s <SRV_Service_URL>
```

If users want to create a custom docker image, they can use the https://github.com/scalar-labs/scalar-admin/tree/scalar-admin-dockerfile#container section